src/styles/variables.ts

```diff
@@ -30,7 +30,8 @@ export const fonts = {
   sansSerif:
     '-apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", Arial, sans-serif',
   serif: 'Georgia, "Times New Roman", Times, serif',
-  monospace: 'Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace, monospace'
+  monospace: 'Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace, monospace',
+  headline: 'GH Guardian Headline, Georgia, "Times New Roman", Times, serif'
```

Comment:
Can we get the Guardian font from `source`?

Reply:
https://theguardian.design/2a1e5182b/p/930d69-overview

Reply:
Thanks @lucymonie 

@mxdvl Maybe this will help: https://theguardian.design/2a1e5182b/p/075107-foundations/t/89a552